src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -556,7 +558,7 @@ import './Menu.css'
             {{/menu-item-description}}
           {{/menu-item}}
           {{#> menu-item-action}}
-            <i class="fas fa-fw fa-ellipsis-v" aria-hidden="true"></i>
+            <i class="fas fa-fw fa-ellipsis-v" aria-label="action 1"></i>
```

Comment:
I've fixed the `aria-label` issue on these examples in another PR. Can you revert these, or do you want me to suggest changes in this PR? The icon should be hidden and the label belongs on the menu-item-action.

Reply:
I merged your PR, will update mine in a few mins.